Source/Core/DiscIO/VolumeDisc.cpp

```diff
@@ -95,6 +95,15 @@ bool VolumeDisc::IsNKit() const
   return ReadSwapped<u32>(0x200, PARTITION_NONE) == NKIT_MAGIC;
 }
 
+bool VolumeDisc::IsTriforceGame() const
+{
+  constexpr u32 BTID_MAGIC = 0x42544944;  // "BTID"
+  return (ReadSwapped<u32>(0x00080000, PARTITION_NONE) == BTID_MAGIC) ||
```

Comment:
Could you explain these hardset read offsets?

Reply:
Those are the locations of the boot.id files in all the triforce games. Previously, it checked using the filesystem, but that broke when I tried to merge it into VolumeGC. This way is also cleaner and shorter, if slightly less ideal.